lighthouse-core/test/gather/fake-driver.js

```diff
@@ -60,7 +60,7 @@ module.exports = {
   },
   beginDevtoolsLog() {},
   endDevtoolsLog() {
-    return require('../fixtures/perflog.json');
+    return require('../fixtures/artifacts/perflog/defaultPass.devtoolslog.json');
```

Comment:
this was big, so just moved over to the artifacts fixtures and moved the two tests using it to point at the new path